test/integration/registered_user_can_view_past_orders_test.rb

```diff
@@ -41,7 +41,7 @@ class RegisteredUserCanViewPastOrdersTest < ActionDispatch::IntegrationTest
 
     order = user.orders.create(quantity: 5, sum: 20)
     visit orders_path
-
+# 
```

Comment:
Incorrect indentation detected (column 0 instead of 4).<br>Trailing whitespace detected.